cluster-scope/overlays/moc/kustomization.yaml

```diff
@@ -27,6 +27,7 @@ resources:
   - ../../base/namespaces/opf-jupyterhub
   - ../../base/namespaces/opf-kafka
   - ../../base/namespaces/opf-monitoring
+  - ../../base/namespaces/openshift-monitoring
```

Comment:
Please include it as a:

```
../../base/clusterroles/...
../../base/clusterrolebindings/...
```

Unless we are deploying a namespace we shouldn't be pulling things into a namespace within the `base/namespaces/` folder and hiding things in there.

I'd rather be this explicit and referred to as "yes, we are really deploying a cluster role".

Reply:
I don't think I can do that, it still needs to populate the `namespace` param, which is done from the namespace directory kustomzation

Reply:
clusterroles and clusterrolebindings don't need the namespace paramater, you have these: 

```
  - ../../../base/clusterrolebindings/metrics
  - ../../../base/clusterroles/metrics
```
In `base/openshift-monitoring/kustomization.yaml`. Remove it from there and include it in the `cluster-scope/overlays/moc/kustomization.yaml`

Reply:
no for the namespace scoped monitoring rbac: https://github.com/operate-first/apps/tree/master/cluster-scope/components/monitoring-rbac

Reply:
I have added the clusterrole/binding in the moc overlay kustomization